src/2_RTB/NormeDiProgetto/content.typ

```diff
@@ -153,7 +153,55 @@ I requisiti trovati avranno un codice univoco con la seguente sintassi:
                 - V -> se di vincolo.
         - *Numero*:
                 per ogni requisito aggiunto il numero viene incrementato.
-     
+
+
+=== Progettazione 
+
+==== Scopo 
+
+L'attività di progettazione verrà svolta dai _Progettisti_, i quali avranno il compito di definire le caratteristiche del prodotto finale, basandosi sui requisiti specificati nel documento _Analisi dei Requisiti_. 
+La fase di progettazione segue l'analisi dei requisiti, nella quale si sono definite le necessità e le aspettative per il prodotto. I Progettisti traducono queste informazioni in una struttura definita, organizzando il sistema in componenti specifici e definendo le interazioni tra di essi. In tal modo, la progettazione costituisce un passo essenziale nel percorso di sviluppo, contribuendo a trasformare i requisiti in un piano tangibile per la creazione del prodotto finale.
```

Comment:
- _Progettisti_ perche' in corsivo? e' termine di glossario? 
- "nella quale sono definite le necessità.."
- "in una struttura definita", cosa si intende per struttura? un'architettura? se si' specificherei cosi'